relay-general/src/store/transactions/processor.rs

```diff
@@ -438,12 +445,42 @@ impl Processor for TransactionsProcessor<'_> {
 
         span.op.get_or_insert_with(|| "default".to_owned());
 
+        if self.scrub_span_descriptions {
+            scrub_span_description(span)?;
+        }
+
         span.process_child_values(self, state)?;
 
         Ok(())
     }
 }
 
+fn scrub_span_description(span: &mut Span) -> Result<(), ProcessingAction> {
+    if span.description.value().is_none() {
+        return Ok(());
+    }
+
+    // let mut scrubbed: Annotated<Value::String> = span.description.clone();
+    let mut scrubbed = span.description.clone();
+
+    // Scrub URLs
+    if scrub_identifiers(&mut scrubbed)? {
```

Comment:
nit: Should we restrict this to specific `span.op`s? It does not make sense to burn CPU cycles on this if we don't expect the description to be a URL.